bin/aws_cleaner.rb

```diff
@@ -26,84 +27,129 @@
 require_relative '../lib/aws-cleaner.rb'
 
 def config(file)
-  YAML.load(File.read(file))
+  YAML.safe_load(File.read(File.expand_path(file)), [Symbol])
 rescue StandardError => e
   raise "Failed to open config file: #{e}"
 end
 
+def logger(config)
+  # By default, STDOUT is used for logging
+  logfile = STDOUT
+
+  if !config[:log].nil? && !config[:log][:file].nil?
+    begin
+      logfile = File.open(File.expand_path(config[:log][:file]), File::CREAT | File::WRONLY | File::APPEND)
```

Comment:
Once you make the above adjustments you can replace `config[:log][:file]` with `logfile`